src/write_outputs/long_duration_storage/write_opwrap_lds_stor_init.jl

```diff
@@ -32,4 +32,52 @@ function write_opwrap_lds_stor_init(path::AbstractString,
     CSV.write(joinpath(path, "StorageInit.csv"),
         dftranspose(dfStorageInit, false),
         header = false)
+
+    # Write storage evolution over full time horizon
+    hours_per_subperiod = inputs["hours_per_subperiod"];
+    t_interior = 2:hours_per_subperiod
+    T = hours_per_subperiod*NPeriods
+    SOC_t = zeros(G, T)
+    stor_long_duration = inputs["STOR_LONG_DURATION"]
+    stor_hydro_long_duration = inputs["STOR_HYDRO_LONG_DURATION"]
+    period_map = inputs["Period_Map"].Rep_Period_Index
+    pP_max = inputs["pP_Max"]
+    e_total_cap = value.(EP[:eTotalCap])
+    v_charge = value.(EP[:vCHARGE])*1000
+    v_P = value.(EP[:vP])*1000
```

Comment:
Is this multiplication intentional? Is it related to the scaling factor? If so, I think we should do something like:
```suggestion
    v_charge = value.(EP[:vCHARGE])
    v_P = value.(EP[:vP])
    if setup["ParameterScale"] == 1
        v_charge *= ModelScalingFactor
        v_P *= ModelScalingFactor
    end
```